tests/server/smoker_test_resources/client_mock_result.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved
+
+import datetime
+from flask.ext.restful import abort
+import json
+import os
+import re
+import socket
+
+
+def generate_unique_file():
+    return datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S-%f')
+
+TMP_DIR = (os.path.dirname(os.path.realpath(__file__)) + '/.tmp')
+PROCESSES = []
+
+
+def rest_api_response(k, **kwargs):
+    if not os.path.exists(TMP_DIR):
+        os.makedirs(TMP_DIR)
+    plugin_list = ['Hostname', 'Uptime', 'Uname']
+    url = 'http://%s:8086' % socket.gethostname()
+    result = {
+        url + '/': about_response,
+        url + '/processes': dict(),
+        url + '/plugins': plugins_response,
+    }
+    if k in result.keys():
+        import ast
```

Comment:
Is there any specifid need for an import in the middle of the code? The best practise is import everything in the beginning.